### PR TITLE
replace pimp with 'rich', 'enrich', 'extension method'

### PR DESCRIPTION
**Pull request by SethTisue**
(this was called to my attention on the “LGBTQ in Tech” Slack)

### CODE REVIEW

**Comment by mgrandi** [comment]
you could wait till a major release when binary compatibility isn't as much as a concern?

**Comment by ktoso** [comment]
We don't anticipate ever having to make such release so far though - compatibility is this libs strongest asset by now

**Comment by ktoso** [comment]
We could keep the old methods but mark them deprecated which would leave this change binary compatible and good-will showing

**Comment by SethTisue** [comment]
he who lives in MiMa-less houses shouldn't throw bincompat stones :-)

**Comment by SethTisue** [comment]
revised version — that also adds MiMa to the build — ready for review

Travis says:

```
[info] spray-json: found 0 potential binary incompatibilities while checking against io.spray:spray-json_2.10:1.3.3 
[info] spray-json: found 0 potential binary incompatibilities while checking against io.spray:spray-json_2.11:1.3.3 
[info] spray-json: found 0 potential binary incompatibilities while checking against io.spray:spray-json_2.12:1.3.3 
```

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: scala
 script:
-  - sbt "+ test"
+  - sbt "+ test" "+ mimaReportBinaryIssues"
```
Comment (ktoso) [inline .travis.yml]: thanks a ton, shocking we didn't do that ;)

**Comment by ktoso** [comment]
Merging this, FYI @jrudolph

**Comment by jrudolph** [comment]
Great, thanks, @SethTisue for taking care of it.

**Comment by SethTisue** [comment]
I see. @ktoso, do you want to add something about it to the 1.3.4 release notes? (I'd do it myself but I don't have admin powers here.)

**Comment by ktoso** [comment]
Yeah I'll add such note. It's only with the specific by name import of previous method that breaks AFAICS

**Comment by ktoso** [comment]
Added a note

**Pull request merged**